From ea2a9f2812d2459ab61baef7e38bd6ccdf880d29 Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 13 Oct 2014 22:30:09 -0400
Subject: [misc] genwqe: Fix format string mismatch in card_debugfs.c

Message-id: <1413239416-7778-22-git-send-email-gduarte@redhat.com>
Patchwork-id: 97555
O-Subject: [RHEL7.1 PATCH BZ 1088491 21/28 v2] misc: genwqe: Fix format string mismatch in card_debugfs.c
Bugzilla: 1088491
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088491
Upstream Status: 32d9dbe37965cb2e7fce47ea836a57cab4ffe5d4

commit 32d9dbe37965cb2e7fce47ea836a57cab4ffe5d4
Author: Masanari Iida <standby24x7@gmail.com>
Date:   Thu Apr 24 12:25:49 2014 +0900

    misc: genwqe: Fix format string mismatch in card_debugfs.c

    Fix two format string mismatch in genwqe_init_debugfs()

    Signed-off-by: Masanari Iida <standby24x7@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/genwqe/card_debugfs.c b/drivers/misc/genwqe/card_debugfs.c
index 50d2096..0a33ade 100644
--- a/drivers/misc/genwqe/card_debugfs.c
+++ b/drivers/misc/genwqe/card_debugfs.c
@@ -348,7 +348,7 @@ int genwqe_init_debugfs(struct genwqe_dev *cd)
  char name[64];
  unsigned int i;
 
- sprintf(card_name, "%s%u_card", GENWQE_DEVNAME, cd->card_idx);
+ sprintf(card_name, "%s%d_card", GENWQE_DEVNAME, cd->card_idx);
 
  root = debugfs_create_dir(card_name, cd->debugfs_genwqe);
  if (!root) {
@@ -454,7 +454,7 @@ int genwqe_init_debugfs(struct genwqe_dev *cd)
  }
 
  for (i = 0; i <  GENWQE_MAX_VFS; i++) {
-  sprintf(name, "vf%d_jobtimeout_msec", i);
+  sprintf(name, "vf%u_jobtimeout_msec", i);
 
   file = debugfs_create_u32(name, 0666, root,
        &cd->vf_jobtimeout_msec[i]);
-- 
1.7.1